sklearn/linear_model/logistic.py

```diff
@@ -770,6 +770,11 @@ def _log_reg_scoring_path(X, y, train, test, pos_class=None, Cs=10,
         the entire probability distribution. Works only for the 'lbfgs'
         solver.
 
+    copy : bool, default True
+        Whether or not to produce a copy of the data. Setting this to
+        True will be useful in cases, when _log_reg_scoring_path is called
```

Comment:
Does this parse correctly? I think sphinx will choke on a leading underscore. Might need backticks.